diff --git a/libxslt/Makefile.in b/libxslt/Makefile.in
index e0a998b..f070844 100644
--- a/libxslt/Makefile.in
+++ b/libxslt/Makefile.in
@@ -375,8 +375,7 @@ libxslt_la_SOURCES = \
        xsltwin32config.h.in            \
        libxslt.h

-@USE_VERSION_SCRIPT_FALSE@LIBXSLT_VERSION_SCRIPT =
-@USE_VERSION_SCRIPT_TRUE@LIBXSLT_VERSION_SCRIPT = $(VERSION_SCRIPT_FLAGS)$(srcdir)/libxslt.syms
+LIBXSLT_VERSION_SCRIPT =
 libxslt_la_LIBADD = $(LIBXML_LIBS) $(EXTRA_LIBS)
 libxslt_la_LDFLAGS = \
                $(WIN32_EXTRA_LDFLAGS)                  \
--
